### DISCUSSION TITLE
How can I stop some controls auto-hiding when video is playing?

### DISCUSSION THREAD

**Discussion by sebastiangug** [muxinc/media-chrome#1038] [Q&A]
I have a mute button that's not added in the bar, but placed with absolute on the bottom-right. 

I've been able to style it with css selectors without any issues but I can't seem to target its inactive state to force it to be visible. 

How would it be best to achieve this: 

```
	media-mute-button {
		background: none;
		position: absolute;
		padding: 2rem;
		bottom: 0;
		right: 0;

		&[data-userinactive] {
			display: block !important;
			visibility: visible !important;
			opacity: 1 !important;
		}
	}

	media-controller[userinactive] media-mute-button {
		display: block !important;
		visibility: visible !important; 
```

**Comment by luwes** [comment]
thanks for the feedback. it seems this is not documented very well.
you can disable auto hiding with the `noautohide` attribute on each direct child of media-controller.

**Comment by sebastiangug** [comment]
thank you, I'll give that a try! I managed to get it working using display:block directly on the `media-mute-button` in the CSS above, but the noautohide attribute seems much cleaner!
